src/app/view/console/ConsoleChessBoardView.ts

```diff
@@ -9,4 +9,10 @@ export class ConsoleChessBoardView implements ChessBoardView {
   }
 
   showSelectedPiece(position: { x: number; y: number }): void {}
+
+  hideAllAvailableMoves(): void {}
+
+  hideSelection(): void {}
+
+  showAvailableMoves(squaresToHighlight: string[]): void {}
```

Comment:
@nowakprojects czy ta klasa jest na wgl potrzebna?

Reply:
Nie jest :) To Wam chcialem pokazac, ze moga byc rozne widoki.

Reply:
@tomdworniczak to w sumie ten plik z folderem można usunąć, com ?